Fix for Java 8

When Travis builds with oraclejdk8 it fails.

I have fixed all but one of the errors through the JavaDoc updates, but I cannot see why this last error is being thrown. Perhaps some fresh eyes on the issue could help. Once this is fixed the project should be nice and ready for Java 8.

Travis Error:

```
[ERROR] Exit code: 1 - javadoc: warning - Error fetching URL: http://java.sun.com/javase/6/docs/api
[ERROR] /home/travis/build/JamoBox/sIRC/src/main/java/com/sorcix/sirc/IrcColors.java:49: error: unexpected text
[ERROR] * {@link "http://www.mirc.com/help/color.txt"}
[ERROR] ^
```

--- a/src/main/java/com/sorcix/sirc/IrcColors.java
+++ b/src/main/java/com/sorcix/sirc/IrcColors.java
@@ -46,7 +46,7 @@ package com.sorcix.sirc;
  * </pre>
  * <p>
  * The mIRC website provides documentation on the color code syntax.
- * {@link "http://www.mirc.com/help/color.txt"}
+ * @see <a href="http://www.mirc.com/help/color.txt">mIRC Colours</a>
  * </p>
  * 
  * @author Sorcix

